docs/resources/vkcs_db_cluster.md

```diff
@@ -83,6 +83,18 @@ The following arguments are supported:
     * `name` - (Required) The name of the capability to apply.
     * `settings` - Map of key-value settings of the capability.
 
+* `restore_point` - Object that represents backup to restore cluster from. It has following attributes:
+    * `backup_id` - (Required) ID of the backup.
+    * `target` - Timestamp of needed backup. You can specify "latest" to use most recent backup.

```

Comment:
probably we must mention here that this works with pitr backups only

Reply:
I think, it should work with all backups, doesn't it? We can create regular backup and specify its id here to restore dbms from it